skill-tree/src/main/resources/application.properties

```diff
@@ -1,5 +1,5 @@
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/${DB_NAME}
+spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:${MYSQL_PORT:3306}/${DB_NAME}
```

Comment:
`:` is this is working like or operator?

Reply:
this works like `||` if we don't have `MYSQL_PORT` use `3306`

Reply:
Got it